onnxruntime/core/providers/openvino/openvino_provider_factory.cc

```diff
@@ -57,6 +57,12 @@ std::unique_ptr<IExecutionProvider> OpenVINOProviderFactory::CreateProvider() {
   std::string so_cache_path = config_options_.GetConfigOrDefault("ep.context_file_path", "").c_str();
   bool so_allow_stripping_qdq = config_options_.GetConfigOrDefault(kOrtSessionOptionsAllowStrippingQDQ, "0") == "1";
 
+  if (!so_allow_stripping_qdq && enable_qdq_optimizer_) {
```

Comment:
If this changes the current default behavior of OpenVINO ep, then we need to consult with maintainers carefully.